libmscore/transpose.cpp

```diff
@@ -665,6 +665,19 @@ void Note::transposeDiatonic(int interval, bool keepAlterations, bool useDoubleA
       score()->undoChangePitch(this, newPitch, newTpc1, newTpc2);
       }
 
+//---------------------------------------------------------
+//   transposeDiatonicAlterations
+//---------------------------------------------------------
+
+void Score::transposeDiatonicAlterations(TransposeDirection direction)
+      {
+      // Transpose current selection diatonically (up/down) while keeping degree alterations
+      // Note: Score::transpose() absolutely requires valid selection before invocation.
+      if (!score()->selection().isNone()) {
+            score()->transpose(TransposeMode::DIATONICALLY, direction, Key::C, 1, true, true, true);
+            }
```

Comment:
Why braces?

Reply:
Guess I'm used to it from my normal 'style' at times.

Reply:
I was used to it too before MuseScore contributions.

Reply:
Yeah, you guys are changing my habits! Hopefully I can continue contributing in the future and won't be off the mark with the guidelines.

Reply:
After some (little bit of) time I guarantee you won't ;-)

Reply:
Won't contribute?! J/k.

Reply:
Does MuseScore really sound this scary? ;-)